### ISSUE TITLE
Add color and background theme preferences

### ISSUE THREAD

**Issue by AdrianoFerrari** [gingko/client#80]

**Comment by brainchild0** [comment]
For me, a large set of fine-grained preferences is not necessary, but it would be very helpful to have some kind of toggle switch into a more subtle set of stylizations.  The blue background with alternating vertical stripes and thick card borders can be distracting.  I would be largely happy with any way to change the display closer to a style of a white background with thin and pale borders.

**Comment by AdrianoFerrari** [comment]
I agree, no large set of preferences. Just a few select "Themes" such as "low contrast, high contrast, inverted, etc".

If you're looking for a workaround for now, you can always use CSS to modify anything you see on the screen. For example, adding this to any card on you tree will make the background gray:
```
<style>
html, body, .buffer {
    background: #cecece;
    border: none;
}
</style>
```

You can see how to change other things by opening the Developer tools (Help menu), and poking around. Here's a tutorial on how to do this, if you're not familiar: ![Get Started With Viewing And Changing CSS](https://developers.google.com/web/tools/chrome-devtools/css)
